### PR TITLE
Update ScalarDB compatibility test matrix

### PR DESCRIPTION
**Pull request by scalar-boney**
Update the ScalarDB compatibility test matrix with all supported databases.

### CODE REVIEW

**Review by feeblefakie — APPROVED** [review]
LGTM.
It's fine for now but the table is too wide to see, so I think you need to think about an alternative format.

File: docs/scalardb-supported-databases.md [hunk 1]
```diff
@@ -6,15 +6,22 @@ Scalar DB supports the following databases and the databases that are compatible
...
-| Scalar DB 2.0.1 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |
+| Scalar DB Version | Cassandra 3.11 | Cassandra 3.0    | Cassandra 2.2   | Cosmos DB | DynamoDB  | MySQL 8.0  | MySQL 5.7  | PostgreSQL 14 | PostgreSQL 13 | PostgreSQL 12 | Oracle 19.0.0.0.ru-2021-04.rur-2021-04.r1 | Oracle 12.2.0.1.ru-2021-04.rur-2021-04.r1 | Oracle 12.1.0.2.v24   | SQL Server 2019 | SQL Server 2017 | SQL Server 2016 | AWS Aurora MySQL 3 (MySQL 8) |  AWS Aurora PostgreSQL 13 | AWS Aurora PostgreSQL 12 | AWS Aurora PostgreSQL 11 |
+|:------------------|:---------------|:-----------------|:----------------|:----------|:----------|:-----------|:-----------|:--------------|:--------------|:--------------|:------------------------------------------|:------------------------------------------|:----------------------|:----------------|:----------------|:----------------|:-----------------------------|:--------------------------|:-------------------------|:-------------------------|
+| Scalar DB 3.4 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |
```
Comment (brfrn169) [inline docs/scalardb-supported-databases.md:18]: @scalar-boney Can you please add the result of the compatibility test for Scalar DB 3.5?

Comment (scalar-boney) [reply docs/scalardb-supported-databases.md:18]: Ok, We will add

Comment (brfrn169) [reply docs/scalardb-supported-databases.md:18]: Since we already released Scalar DB 3.5, can you please do that by EOD today?

Comment (scalar-boney) [reply docs/scalardb-supported-databases.md:19]: @brfrn169 san,
Sorry for the delay.
Compatibility matrix updated with Scalar DB 3.5.

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**